index.html

```diff
@@ -291,6 +291,125 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <section id="mycontact" class="footer">
+      <h4 class="invitation">
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h4>
+
+      <form action="https://formspree.io/f/mqknkkgn" method="POST" class="form-element">
+        <ul>
+          <li class="full-name">
+            <label for="name"></label>
+            <input
+              class="text"
+              type="text"
+              id="name"
+              name="full_name"
+              placeholder="Full name"
+              minlength="3"
+              maxlength="30"
+              required
+            />
+            <img class="break" src="assets/images/Container.svg" alt="line" />
+          </li>
+
+          <li class="email">             
+            <label for="mail"></label>
+            <input
+              class="text"
+              type="email"
+              id="mail"
+              name="user_email"
+              placeholder="Email address"
```

Comment:
- [ ] [**OPTIONAL**] I realized that you have built your form inputs as list items(they are wrapped in the `ul` tag and each is wrapped into the `li` tag). While I don't know the reason why you have done so, I would like to tell you that it is possible that **you can remove the input tags from the list tag and still achieve the same results as it is the Figma design given💡.** You will only need to add the correct styles. In addition, It is always a good practice to keep your codes simple and clean so that other developers can understand them.[ Learn more about it ](https://www.interaction-design.org/literature/topics/keep-it-simple-stupid). That is why I would suggest **you not wrap them in list tags**.
- [ ] [**OPTIONAL**] Another point I would say is that your form inputs don't need the `labels`  referring to the Figma Design you are using. It would be better if you ignore them 😺